lens.cabal

```diff
@@ -348,6 +335,8 @@ library
 
   default-language: Haskell2010
 
+  x-docspec-options: --extra-package=simple-reflect --extra-package=deepseq
```

Comment:
I wonder if it would be cleaner to be able to say `--extra-packages="simple-reflect deepseq"`. I could imagine that having to duplicate `--extra-package=` for each dependency would become annoying.

Reply:
due how parsing in `.cabal` files work it would need to be
```
"--extra-package=simple-reflect deepseq"
```

Which I'm sure people won't get right. (Though it's consistent with `--ghc-options` etc.)

An alterntive is to make
```
x-docspec-extra-packages: simple-reflect deepseq
```
possible, but for now I like the open-word of `x-doctest-options`. (As said, I expect `lens` to be the only one needing two arguments!)